hphp/runtime/vm/jit/tc-recycle.cpp

```diff
@@ -137,20 +139,33 @@ void clearProfCaller(TCA toSmash, const Func* func, int numArgs,
 void clearTCMaps(TCA start, TCA end) {
   auto const profData = jit::profData();
   while (start < end) {
-#if defined(__powerpc64__)
+    bool isBranch, isNop, isCall;
+#if defined(__aarch64__)
```

Comment:
Since you're adding these bools, you should just convert these `#if`s to be regular `if ()`s instead, switching on architecture.